src/validate/editor.js

```diff
@@ -0,0 +1,20 @@
+'use babel'
+
+import throwIfFail from './throw'
+
+export const isValidPoint = (textBuffer, line, column) => {
+  // Clip the given point to a valid one
+  const validPoint = textBuffer.clipPosition([line, column])
+  // Compare to original
+  return validPoint.isEqual([line, column])
+}
+
+export const throwIfInvalidPoint = (tb, line, col) => throwIfFail(
+  (`${line}:${col} isn't a valid point!`),
```

Comment:
Why is this wrapped in `()`?

Reply:
Artifact from it's old context. Shouldn't be there.